dotCMS/src/main/java/com/dotcms/rest/api/v1/system/StartupResource.java

```diff
@@ -0,0 +1,122 @@
+package com.dotcms.rest.api.v1.system;
+
+import com.dotcms.repackage.javax.ws.rs.*;
+import com.dotcms.repackage.javax.ws.rs.core.Context;
+import com.dotcms.repackage.javax.ws.rs.core.MediaType;
+import com.dotcms.repackage.javax.ws.rs.core.Response;
+import com.dotcms.repackage.org.glassfish.jersey.server.JSONP;
+import com.dotcms.rest.InitDataObject;
+import com.dotcms.rest.ResponseEntityView;
+import com.dotcms.rest.WebResource;
+import com.dotcms.rest.annotation.NoCache;
+import com.dotcms.rest.api.v1.authentication.ResponseUtil;
+import com.dotcms.util.CollectionsUtils;
+import com.dotcms.util.ReflectionUtils;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.UserAPI;
+import com.dotmarketing.exception.DoesNotExistException;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.startup.StartupAPI;
+import com.dotmarketing.util.Logger;
+import com.google.common.annotations.VisibleForTesting;
+import com.liferay.portal.model.User;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Path("/v1/startuptasks")
+public class StartupResource {
```

Comment:
I would name it `StartupTaskResource` is very specific to startup tasks and just `Startup` could create confusion about the purpose of the class.